### DISCUSSION TITLE
Small Idea about  the beacon mode in the radio (TX).

### DISCUSSION THREAD

**Discussion by slavikby** [egzumer/uv-k5-firmware-custom#298] [Ideas]
There is an idea about the beacon mode in the radio station. Is it possible to periodically broadcast, for example, a tone signal?

**Comment by ErikS-web** [comment]
Can also be done with external hardware on the PTT-signal and must be used in combination with a callsign.

For HAM-use it requires a registration to be active as a beacon.
